b2sdk/account_info/sqlite_account_info.py

```diff
@@ -29,7 +29,9 @@
 B2_ACCOUNT_INFO_PROFILE_NAME_REGEXP = re.compile(r'[a-zA-Z0-9_\-]{1,64}')
 XDG_CONFIG_HOME_ENV_VAR = 'XDG_CONFIG_HOME'
 
-DEFAULT_ABSOLUTE_MINIMUM_PART_SIZE = 5000000  # this value is used ONLY in migrating db, and in v1 wrapper, it is not
+DEFAULT_ABSOLUTE_MINIMUM_PART_SIZE = (
+    5000000  # this value is used ONLY in migrating db, and in v1 wrapper, it is not
+)
```

Comment:
Less readable to me

Reply:
I think it's just because of width limit being too low? @vbaltrusaitis-reef

Reply:
Yes, black formats it back to the original version with `line-length=120`. Whether 120 is a good number is another question. :)